File: src/dto/diapograph.dto.ts
import { DiapositivaDTO } from './diapositiva.dto';
import { GraphDTO } from './graph.dto';


export class  DiapoGraphDTO{
  id: number;
  dimensione: string;
  posizione: string;
  diapositiva: DiapositivaDTO;
  graph: GraphDTO;

  constructor(diapositiva: DiapositivaDTO,id?: number, dimensione?: string, posizione?: string,  graph?: GraphDTO){
    this.id = id ? id:0;
    this.dimensione = dimensione ? dimensione : "25_25";
    this.posizione = posizione ? posizione : "1_1";
    this.diapositiva = diapositiva;
    this.graph = graph ? graph : new GraphDTO(null);
  }
}

File: Presentazione/Presentazione/DTO/DiapographDTO.cs
using Presentazione.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Presentazione.DTO
{
    public class DiapographDTO
    {
        public long id { get; set; }

        public string dimensione { get; set; }

        public string posizione { get; set; }

        public DiapositivaDTO diapositiva { get; set; }

        public long idGraph { get; set; }
    }
}
